arches_querysets/models.py

```diff
@@ -125,6 +125,13 @@ def save(self, *, request=None, index=True, force_admin=False, **kwargs):
         ):
             raise ValidationError(_("Graph Has Different Publication"))
 
+        # Emulate some computations trapped on Resource.save()
+        # _save_aliased_data() will call super().save(), i.e. ResourceInstance.save()
+        if arches_version >= (8, 0) and not self.resource_instance_lifecycle_state_id:
```

Comment:
oh, does this need to be here? If possible I'd like to keep that logic wrangled into one place

Reply:
I'm taking a stab at that ☝️

Reply:
Yeah, this was also missing the lifecycle handlers call, so we'll need to add this to this workaround if we don't do that refactor in core.

Reply:
https://github.com/archesproject/arches/pull/12323